python/lsst/ts/externalscripts/auxtel/make_latiss_calibrations.py

```diff
@@ -71,6 +73,21 @@ def instrument_name(self):
         """String with instrument name for pipeline task"""
         return "LATISS"
 
+    @property
+    def pipeline_instrument(self):
+        """String with instrument name for pipeline yaml file"""
+        return "Latiss"
+
+    @property
+    def detectors(self):
+        """Array with detector IDs"""
+        return self._detectors
+
+    @property
+    def n_detectors(self):
+        """Number of detectors"""
+        return self._n_detectors
```

Comment:
Consider removing both `self._detectors` and `self._n_detectors`, and just returning the constant values. e.g.;

```
@property
def n_detectors(self):
    return 1

@property
def detectors(self):
    return [0]
```